### PR TITLE
feat: add clearKey method

### PR DESCRIPTION
**Pull request by ProbablePrime**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +140,17 @@ sharp(path.resolve(__dirname, 'github_logo.png'))
...
 
+#### <a name="clear-key"></a> `> streamDeck.clearKey(keyIndex) -> undefined`
+
+Synchronously clears the given `keyIndex`'s screen.
+
+##### Example
+
+```javascript
+// Clear the third button from the left in the first row.
+streamDeck.clearKey(2, buffer);
```
Comment (Lange) [inline README.md]: Looks like the `, buffer` part is left over from a copy/paste. Once that's removed, this will be good to merge.

Comment (ProbablePrime) [inline README.md]: yarp!

**Comment by Lange** [comment]
lgtm 👌

**Comment by ProbablePrime** [comment]
Build failure is #5

**Pull request merged**